nevergrad/benchmark/test_xpbase.py

```diff
@@ -19,7 +19,7 @@
 from . import xpbase
 
 
-DESCRIPTION_KEYS = {"seed", "elapsed_time", "elapsed_budget", "loss", "optimizer_name", "pseudotime",
+DESCRIPTION_KEYS = {"split", "seed", "elapsed_time", "elapsed_budget", "loss", "optimizer_name", "pseudotime",
```

Comment:
```suggestion
DESCRIPTION_KEYS = {"seed", "elapsed_time", "elapsed_budget", "loss", "optimizer_name", "pseudotime",
```
This is already in ARTIFICIAL_KEYS